From 6cc3708dbc34f3652c107770b4d5fed02304df38 Mon Sep 17 00:00:00 2001
From: "Eric S. Raymond" <esr@thyrsus.com>
Date: Sat, 3 Dec 2016 14:15:17 -0500
Subject: [PATCH 162/268] Address GitLab issue #188: asciidoc puts <span>'s in
 page <title>'s

This shouldn't actually be necessary - we're working around a bug in
Chrome and Firefox here.
---
 docs/msyslog.txt      | 2 +-
 docs/ntpd.txt         | 2 +-
 docs/ntpdig.txt       | 2 +-
 docs/ntpfrob.txt      | 2 +-
 docs/ntpleapfetch.txt | 2 +-
 docs/ntpq.txt         | 2 +-
 docs/ntptime.txt      | 2 +-
 docs/ntptrace.txt     | 2 +-
 docs/ntpviz.txt       | 2 +-
 docs/ntpwait.txt      | 2 +-
 docs/prefer.txt       | 2 +-
 11 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/docs/msyslog.txt b/docs/msyslog.txt
index ee96813..29caa56 100644
--- a/docs/msyslog.txt
+++ b/docs/msyslog.txt
@@ -1,4 +1,4 @@
-= +ntpd+ System Log Messages =
+= ntpd System Log Messages =
 
 [cols="10%,90%",frame="none",grid="none",style="verse"]
 |==============================
diff --git a/docs/ntpd.txt b/docs/ntpd.txt
index ff899c1..9e927c4 100644
--- a/docs/ntpd.txt
+++ b/docs/ntpd.txt
@@ -1,4 +1,4 @@
-= +ntpd+ - Network Time Protocol (NTP) Daemon =
+= ntpd - Network Time Protocol (NTP) Daemon =
 
 [cols="10%,90%",frame="none",grid="none",style="verse"]
 |==============================
diff --git a/docs/ntpdig.txt b/docs/ntpdig.txt
index 7ce28ff..931af4b 100644
--- a/docs/ntpdig.txt
+++ b/docs/ntpdig.txt
@@ -1,4 +1,4 @@
-= +ntpdig+ - Simple Network Time Protocol (SNTP) Client =
+= ntpdig - Simple Network Time Protocol (SNTP) Client =
 
 [cols="10%,90%",frame="none",grid="none",style="verse"]
 |==============================
diff --git a/docs/ntpfrob.txt b/docs/ntpfrob.txt
index 188e52d..e952129 100644
--- a/docs/ntpfrob.txt
+++ b/docs/ntpfrob.txt
@@ -1,4 +1,4 @@
-= `ntpfrob` - frob the local clock hardware =
+= ntpfrob - frob the local clock hardware =
 
 [cols="10%,90%",frame="none",grid="none",style="verse"]
 |==============================
diff --git a/docs/ntpleapfetch.txt b/docs/ntpleapfetch.txt
index 52484d0..7a5396b 100644
--- a/docs/ntpleapfetch.txt
+++ b/docs/ntpleapfetch.txt
@@ -1,4 +1,4 @@
-= +ntpleapfetch+ - fetch and manage local leap file =
+= ntpleapfetch - fetch and manage local leap file =
 
 //FIXME: Image duplicates the one used for ntpfrob
 [cols="10%,90%",frame="none",grid="none",style="verse"]
diff --git a/docs/ntpq.txt b/docs/ntpq.txt
index bedc73d..8fff9c6 100644
--- a/docs/ntpq.txt
+++ b/docs/ntpq.txt
@@ -1,4 +1,4 @@
-= +ntpq+ - standard NTP query program =
+= ntpq - standard NTP query program =
 
 [cols="10%,90%",frame="none",grid="none",style="verse"]
 |==============================
diff --git a/docs/ntptime.txt b/docs/ntptime.txt
index 61a6a97..f0372dd 100644
--- a/docs/ntptime.txt
+++ b/docs/ntptime.txt
@@ -1,4 +1,4 @@
-= `ntptime` - read and set kernel time variables =
+= ntptime - read and set kernel time variables =
 
 [cols="10%,90%",frame="none",grid="none",style="verse"]
 |==============================
diff --git a/docs/ntptrace.txt b/docs/ntptrace.txt
index 02dcbba..94d3dfa 100644
--- a/docs/ntptrace.txt
+++ b/docs/ntptrace.txt
@@ -1,4 +1,4 @@
-= `ntptrace` - trace a chain of NTP servers back to the primary source =
+= ntptrace - trace a chain of NTP servers back to the primary source =
 
 [cols="10%,90%",frame="none",grid="none",style="verse"]
 |==============================
diff --git a/docs/ntpviz.txt b/docs/ntpviz.txt
index 22cce8a..846b8c5 100644
--- a/docs/ntpviz.txt
+++ b/docs/ntpviz.txt
@@ -1,4 +1,4 @@
-= +ntpviz+ - make data visualizations from statistics files =
+= ntpviz - make data visualizations from statistics files =
 
 [cols="10%,90%",frame="none",grid="none",style="verse"]
 |==============================
diff --git a/docs/ntpwait.txt b/docs/ntpwait.txt
index f0eb54a..05d8545 100644
--- a/docs/ntpwait.txt
+++ b/docs/ntpwait.txt
@@ -1,4 +1,4 @@
-= +ntpwait+ - wait until ntpd is in synchronized state =
+= ntpwait - wait until ntpd is in synchronized state =
 
 include::includes/ntpwait-body.txt[]
 
diff --git a/docs/prefer.txt b/docs/prefer.txt
index 14829c4..c59bcf2 100644
--- a/docs/prefer.txt
+++ b/docs/prefer.txt
@@ -1,4 +1,4 @@
-= Mitigation Rules and the +prefer+ Keyword =
+= Mitigation Rules and the prefer Keyword =
 
 [cols="10%,90%",frame="none",grid="none",style="verse"]
 |==============================
-- 
2.7.4

